### PR TITLE
AEX-2: Final

### PR DESCRIPTION
**Pull request by shekhar-shubhendu**

### CODE REVIEW

**Comment by shekhar-shubhendu** [comment]
closes #22

File: AEXS/aex-2.md [hunk 1]
```diff
@@ -23,28 +23,17 @@ By defining the standard way of communication between SDK(aepps) and Wallet we w
...
-
-- `ping/pong`: general ping/pong messages to check liveness. Implementation of this method is not mandatory for wallets communicating over a transport layer that have native support for liveness check.
-
-    ##### Parameters
+### Naming Convention
 
-    `Object`
-
-  - `id` - A unique identifier, must conform to the [UUID v4 standards](https://tools.ietf.org/html/rfc4122#page-14)
+This document tries to closely follow `who.what.how` naming convention for JSON-RPC methods(wherever possible).
```
Comment (davidyuk) [inline AEXS/aex-2.md:28]: Do you have any explanation/motivation/reasoning of this naming convention? Maybe we can try to make it a separate standard to use it in all AEXs?

Comment (shekhar-shubhendu) [reply AEXS/aex-2.md:28]: as mentioned earlier on several occasions we follow the naming convention as per the naming convention used earlier in aeternity. you can find more details here https://github.com/aeternity/protocol/tree/master/channels#terms

Comment (davidyuk) [reply AEXS/aex-2.md:28]: Also, there is a missed space before "("

**Review by thepiwo — APPROVED** [review]

**Review by davidyuk — CHANGES_REQUESTED** [review]
I think Final status should be delayed until corresponding issues get resolved: https://github.com/aeternity/AEXs/issues/51, https://github.com/aeternity/AEXs/issues/50, https://github.com/aeternity/AEXs/issues/49, https://github.com/aeternity/AEXs/issues/48, https://github.com/aeternity/AEXs/issues/47, https://github.com/aeternity/AEXs/issues/46, https://github.com/aeternity/AEXs/issues/44, https://github.com/aeternity/AEXs/issues/38, https://github.com/aeternity/AEXs/issues/29, https://github.com/aeternity/AEXs/issues/26
Their resolving may lead to breaking changes.

**Comment by thepiwo** [comment]
Thanks for all the feedback and discussion points. I do agree with you that we will have to delay a final version due to this.

**Pull request closed without merging**